itk-module.cmake

```diff
@@ -1,10 +1,9 @@
 set(DOCUMENTATION "The modules provides filters may be particularly useful for
 ultrasound image reconstruction and analysis.")
 
-# A library is only created when FFTW support is ON
 set(_fft_depends ITKFFT)
-if(ITK_USE_FFTWF OR ITK_USE_FFTWD)
-  set(_fft_depends ITKFFT)
+if(ITKUltrasound_USE_clFFT)
+  list(APPEND _fft_depends ITKGPUCommon)
```

Comment:
Is the `ITKGPUCommon` dependency needed?

Reply:
It is, it sets up include directories for OpenCL.